src/content/en/2020/privacy.md

```diff
@@ -12,12 +12,166 @@ translators: []
+|mobile|	doubleclick.net|	test_cookie|32%
+|mobile|	doubleclick.net	|IDE|	21%
+|mobile|	facebook.com|	fr|	10%
+|mobile|	youtube.com|	VISITOR_INFO1_LIVE|10%
+|mobile	|youtube.com|	YSC|10%
+|mobile|	google.com|	NID|10%
+|mobile|	youtube.com	|GPS|	8%
+|mobile|	doubleclick.net	|DSID|	7%
+|mobile|	yandex.ru|	yandexuid|	6%
+|mobile|	yandex.ru|	i|	6%
+
+<figcaption>{{ figure_link(caption="Top cookies", sheets_gid="732942035", sql_file="top100_cookies_set_from_header.sql") }}</figcaption>
+</figure>
+
+As found previously, Google’s  tracking domain ‘doubleclick.net’  sets cookies on roughly one fourth of websites on a mobile client and one third of all  websites on a desktop client. Again, nine out of the ten most popular cookies on desktop client and 7 on mobile are set by a Google domain. This is a lower bound for the amount of websites the cookie is set on, since we are only conting cookies set via an HTTP header, a large amount of tracking cookies is set by using third-party scripts.
+
+### Fingerprinting
+
+Another widely-used tracking technique is fingerprinting, which consists of collecting different kinds of information about the user with the goal of building a unique “fingerprint” for them. Different types of fingerprinting are used on the web by trackers. Browser fingerprinting uses characteristics specific to the browser of the user, relying on the fact that the chance of another user having the exact same browser information is fairly small. In our crawl, we examined the presence of  the [FingerprintJS](https://fingerprintjs.com/) library, which provides browser fingerprinting as a service.  
+
+{{ figure_markup(
+  image="websites_with_fingerprint.js_library.png",
+  caption="Websites using FingerprintJS",
+  description="This figure shows the percentage of websites using FingerprintJS.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQJMtHv0Y1JcQJkcVyrqBk9dsujZeDtOZEv7uvE0xM2VrQSuTUDFya41TeRlTZDDe2rWmHwDghW3Dev/pubchart?oid=1556252953&format=interactive",
+  sheets_gid="222110824",
+    sql_file="percent_of_websites_with_fingerprinting.sql "
+  )
+}}
+Although the library is present on less than 1% of websites, the main issue with fingerprinting is its persistent nature.  Furthermore, FingerprintJS is not the only attempt at fingerprinting.  Other libraries, tools and native code can also serve this purpose.
+
+## Consent Management Platforms
+Cookie consent banners are being used by websites for years now, to allow users to specify their third-party cookie choices as well as increase transparency towards them. While a lot of websites opt for using their own implementation of cookie banners, third-party solutions called ‘Consent Management Platforms’ have recently emerged. The platforms provide an easy way for websites to collect user’s consent for different types of cookies. We see that 4.4% of websites use some consent management platform  to provide cookie choices on desktop clients, and 4% on mobile clients. 
```

Comment:
Not all banners allow cooking choice. Some are still (dubiously!) going with "By using this website you consent..."

```suggestion
Cookie consent banners have become common now increase transparency towards cookies and often allowing users to specify their cookie choices. While a lot of websites opt for using their own implementation of cookie banners, third-party solutions called ‘Consent Management Platforms’ have recently emerged. The platforms provide an easy way for websites to collect user’s consent for different types of cookies. We see that 4.4% of websites use some consent management platform to provide cookie choices on desktop clients, and 4.0% on mobile clients. 
```

Reply:
@ydimova did you see this comment?

Reply:
Ditto.